utils/text/transformers.py

```diff
@@ -129,8 +129,8 @@ def seo_tokenize(title, lower=True, preserve_unicode=False):
     def _repl(matchobj):
         c = matchobj.group(0)
         if is_ascii(c) or is_ascii_extended(c):
-            # it is ASCII, but not one of the accepted ASCII characters
-            replaced_c = ''
+            # it is an accepted ASCII character
```

Comment:
I think the correct fix is to leave this block alone, but move `if preserve_unicode` to the first block, and make this an `elif`

Reply:
could you also add a comment after line 106:
```
`preserve_unicode` will preserve Unicode and extended ASCII characters if `True`
```

Reply:
actually, what we need to do is add another parameter _before_ `preserve_unicode`, called `preserve_ascii_extended`


and change to:

```
if is_ascii_extended(c):
    replaced_c = c if preserve_ascii_extended else ''
elif is_ascii(c):
    # it is ASCII, but not one of the accepted ASCII characters
    replaced_c = ''
else:
    replaced_c = c if preserve_unicode else ''
```